dom-event-delegation.tar/dom-event-delegation/main.js

```diff
@@ -0,0 +1,8 @@
+const taskList = document.querySelector('.task-list'); // select the button element
+
+taskList.addEventListener('click', function(event){
+  console.log('event.target:',event.target);
+  console.log('event.target.tagName:',event.target.tagName);
+  console.log(event.target.closest('.task-list-item'));
+  console.log(event.target.remove(event.target.closest('.task-list-item')));
```

Comment:
these should not be inside `console.log` method calls.

In addition, before calling `event.target.closest()` and `event.target.remove()` there should be an `if` statement to make sure that `event.target.tagName` is equal to `'BUTTON'`